### PR TITLE
Exclude 1.4

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
Can't figure out how to yank version 1.4 of flyteidl (which was published in error and already yanked on pypi).

Also signal route is wrong.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by wild-endeavor** [comment]
will be put into the big refactor pr
